ui/components/multichain/asset-picker-amount/index.scss

```diff
@@ -1,17 +1,87 @@
 .asset-picker-amount {
-  &__swap-button {
-    background: none;
+  // balance text elements
+  &__balance > *,
+  &__balance > div > * {
+    color: var(--color-text-alternative);
+    font-size: small;
+    padding: 0;
   }
 
+  // input for NFTs
+  &__input-nft {
+    &:focus-within {
+      border-color: inherit;
+    }
+
+    // input field itself
+    & div > div > input {
+      text-align: right;
+    }
+
+    flex-shrink: 1;
+    box-sizing: border-box;
+    border-width: 1px;
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+
+    // 4 padding + 1 border
+    margin: -5px -5px -5px 0;
+    padding: 24px 16px 24px 16px;
+  }
+
+  // input for tokens
   &__input {
-    padding: 0 0 0 8px;
-    width: 100%;
+    & > div {
```

Comment:
Can we use classnames instead of `>`?

Reply:
Sure! Still keeping it nested here for clarity.